### ISSUE TITLE
Get 'mentors' command working. 

### ISSUE THREAD

**Issue by bbugyi200** [HackRU/SlackRU#4]
- [ ] Get Twilio Login Information
- [ ] Figure out a way to set up Flask Development Server (use pythonanywhere.com if no easier option is evident). `serverside.py` will reside and run on this server.
- [ ] Verify functionality.
- [ ] Write Unit Tests to make future tests easier. :)

**Comment by bbugyi200** [comment]
We cannot start to work on this until questions #8 and #9 are answered.

**Comment by bbugyi200** [comment]
This is no longer relevant. (reference #9)
